tiles-generation/docker/vector-tile/modified-s5cmd/cp.go

```diff
@@ -0,0 +1,868 @@
+//  Copyright OpenSearch Contributors
+//  SPDX-License-Identifier: Apache-2.0
+
+// In order to support add metadata 'Content-Encoding=gzip' and 'Content-Type=application/x-protobuf' on copy command,
+// modified this file from https://github.com/peak/s5cmd for docker image build.
+
+package command
+
+import (
```

Comment:
File name as cp.go what does it stand for, can we do better at naming the files.

Reply:
The file name is accordingly with original name in s5cmd, it might be more confused if we not use the original name.